frontend_react/src/container/Contact.jsx

```diff
@@ -1,9 +1,155 @@
+    console.log(formState);
+  };
+
+  useEffect(() => {
+    if (notification) {
+      const timer = setTimeout(() => {
+        setNotification('');
+      }, 5000);
+      return () => clearTimeout(timer);
+    }
+  }, [notification]);
+
+  const sendEmail = (e) => {
+    e.preventDefault();
+    setLoading(true);
+    emailjs.sendForm('service_bkpc7ag', 'template_ophm5cg', form.current, 'qbFMLvC8bIjmsP9iB')
+      .then(() => {
+        setNotification('Email sent successfully');
+        setLoading(false);
+        setFormState({
+          name: '',
+          email: '',
+          message: '',
+        });
+      })
+      .catch(() => {
+        setNotification('Error sending email');
+      });
+  };
 
-function Contact() {
   return (
-    <div>Contact</div>
+    <motion.div
+      initial={{ opacity: 0 }}
+      whileInView={{ opacity: [0, 1] }}
+      hidden={{ opacity: 1 }}
+      viewport={{ once: false, amount: 0.25 }}
+      id="CONTACT"
+      className={`flex flex-col md:flex-row md:h-viewport-height pt-[110px] md:gap-20 md:px-[100px] lg:px-[200px] px-5  overflow-y-auto
+       ${theme === 'light' ? 'bg-bgContact' : 'bg-bgContactDark'} bg-center bg-no-repeat bg-cover`}
+    >
+      <div className="flex-1 justify-center items-center">
+        <div className="flex flex-col md:flex-row justify-center items-center md:gap-2">
+          <div className="flex w-[86px] h-0.5 bg-primary mb-2" />
+          <h1 className="text-[36px] md:text-[42px] font-medium pb-5">Got a project? Let’s talk</h1>
+        </div>
+        <h1 className="hidden md:flex">
+          <span className="pb-4 text-secondary font-bold italic">
+            “Plan, take Massive Action and Fail Forward.  –Richard Leung Woo-Gabriel ”
+          </span>
+        </h1>
+        <div className="flex flex-row justify-center items-center gap-4 pb-10">
+          <motion.a whileHover={{ scale: 1.1 }} className="hover:shadow-primary shadow-md rounded-full" href="https://www.facebook.com/richard.l.gabriel.3/" target="_blank">
+            <TiSocialFacebookCircular size={35} />
+          </motion.a>
+          <motion.a whileHover={{ scale: 1.1 }} className="hover:shadow-primary shadow-md rounded-full" href="https://www.linkedin.com/in/richard-leung-woo-gabriel-0b0822232/" target="_blank">
+            <TiSocialLinkedinCircular size={35} />
+          </motion.a>
+          <motion.a whileHover={{ scale: 1.1 }} className="hover:shadow-primary shadow-md rounded-full" href="https://github.com/leungwoo" target="_blank">
+            <ImGithub size={30} />
+          </motion.a>
+        </div>
+        <img src={contactProject} alt="contact" className=" object-cover max-h-[400px] md:max-h-[500px] rounded-full pb-5" />
+      </div>
+      <form onSubmit={sendEmail} ref={form} className=" flex-1 mx-auto max-w-md mb-2">
+        {notification && <div className="text-primary font-medium justify-center items-center">{notification}</div>}
+        <div className="mb-4">
```

Comment:
Instead of using mb-4 on each child, you can use gap properties on the parent element. That will ensure proper spacing between its children:
```javascript
<form className="flex-1 flex flex-col gap-4">
  <ElementOne />
  <ElementOne />
  <ElementOne />
</form>
```